### ISSUE TITLE
mypy cannot find client or resource without implicit_reexport

### ISSUE THREAD

**Issue by youngbob** [vemel/mypy_boto3_builder#34]
**Describe the bug**
When mypy is run with implicit_reexport=False (either on the command line or config), it fails to find "client" and "resource" exports from boto3 module.

https://mypy.readthedocs.io/en/stable/config_file.html#miscellaneous-strictness-flags

**To Reproduce**
1. Install 'boto3-stubs["lambda"]'
2.  Run mypy --no-implicit-reexport on the following code sample

```python
import boto3

client = boto3.client("lambda")
```
3. Actual mypy output

```
test.py:3: error: Module has no attribute "client"
```

**Expected mypy-output**

```
Success: no issues found in 1 source file
```

**Additional context**
mypy 0.770
boto3 1.14.18
boto3-stubs 1.14.17.0

**Comment by vemel** [comment]
Never actually used it. I should check `mypy` documentation.

**Comment by vemel** [comment]
Finally fixed in `1.14.54.1`. Please let me know if it works as it should.

**Comment by youngbob** [comment]
Tested in 1.14.56.0 and seems to work. Thanks for the fix!
